src/components/ListingDetails/index.tsx

```diff
@@ -56,7 +57,12 @@ const caseFields: ListingField<CaseListing>[] = [
   // Remote/In Person
   {
     label: 'Remote/In Person',
-    getValue: data => (data.is_remote ? 'Remote' : 'In Person'),
+    getValue: data => {
+      if (data.is_remote === null || data.is_remote === undefined) {
+        return 'Not Available';
+      }
+      return data.is_remote ? 'Remote' : 'In Person';
+    },
   },
   // Adjudicating Agency
   {
```

Comment:
do you think we can update the other "N/A" to "Not available" on this page (i.e., the country, adjudicating agency, etc)?? 

Do you think you could also update the `ListingCard` in this PR such that we're not defaulting to `In Person` for the location? let's display "Not Available" there as well (similar to what we did with the `ListingDetails`)

Reply:
yes sg!